3rdparty/python/lockfiles/user_reqs.txt

```diff
@@ -11,13 +11,15 @@
 #   "generated_with_requirements": [
 #     "PyYAML<7.0,>=6.0",
 #     "ansicolors==1.1.8",
+#     "checksumdir==1.1.7",
 #     "chevron==0.14.0",
 #     "fasteners==0.16.3",
 #     "freezegun==1.1.0",
 #     "humbug==0.2.7",
 #     "ijson==3.1.4",
+#     "numpy==1.14.5",
```

Comment:
This stood out as nonsensical. But, it turns out:
```
$ git blame testprojects/3rdparty/python/requirements.txt 
93934642ff (Danny McClanahan 2019-10-14 00:10:54 -0500 1) checksumdir==1.1.7
93934642ff (Danny McClanahan 2019-10-14 00:10:54 -0500 2) numpy==1.14.5
```
But `build-support/bin/generate_all_lockfiles.sh` and `build-support/bin/_generate_all_lockfiles_helper.py` and ` have not been changed since late last year, whereas the locks were re-generated on January 11th by @asherf. So this may be a bug in the lock infra somehow introduced since then?

Reply:
I have no clue if the inclusion of these testprojects deps in our lock is harmful or not. I'll let reviewers speak up and determine if a fix is needed in which case I'll participate in or wait for that before moving forward with this PR.